clients/cli/src/main/java/org/apache/gravitino/cli/commands/SetCatalogProperty.java

```diff
@@ -74,4 +74,10 @@ public void handle() {
 
     System.out.println(catalog + " property set.");
   }
+
+  @Override
+  public Command validate() {
+    checkProperty(property, value);
+    return this;
+  }
```

Comment:
You could have checkProperty return this and change this to just `return checkProperty(property, value);` but either is probably fine